src/components/modal/examples/ModalCompositeConnectedExamples.tsx

```diff
@@ -13,14 +19,45 @@ const modalId: string = 'modal-composite-connected';
 const secondModalId: string = 'second-modal-composite';
 const insideModalId: string = 'inside-modal';
 
-const mapStateToProps = () => ({});
-
 const mapDispatchToProps = (dispatch: (action: IReduxAction<IModalActionPayload>) => void): IModalExamplesProps => ({
     openModal: (id: string) => dispatch(openModal(id)),
     closeModal: (id: string) => dispatch(closeModal(id)),
 });
 
-@ReduxConnect(mapStateToProps, mapDispatchToProps)
+const lorem = loremIpsum({count: 10});
+
+@ReduxConnect(undefined, mapDispatchToProps)

```

Comment:
You could define the mapDispatchToProps as an object instead (see [react-redux's doc](https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object) about it, it is actually the recommended way to use mapDispatchToProps).
```suggestion
@ReduxConnect(null, {openModal, closeModal})
```
And you can remove the `mapDispatchToProps` function declaration.